Assignment3/riverita98-y160305/asignment3.txt

```diff
@@ -0,0 +1,274 @@
+﻿1- Get all the properties that can be applied to instances of the 
+Politician class (<http://dbpedia.org/ontology/Politician>)
+
+select ?properties where { 
+    ?x a  <http://dbpedia.org/ontology/Politician> .
+?x ?properties ?z }
+
+
+RESULT:
+_______________________________________________________
+
+properties
+
+rdf:type [http]
+rdfs:label [http]
+rdfs:comment [http]
+owl:differentFrom [http]
+rdfs:seeAlso [http]
+owl:sameAs [http]
+dbpedia:ontology/deathPlace [http]
+dbpedia2:deathPlace [http]
+dbpedia:ontology/deathDate [http]
+dbpedia:ontology/birthPlace [http]
+dbpedia:ontology/birthDate [http]
+<http://purl.org/dc/terms/subject> [http]
+dbpedia:ontology/wikiPageID [http]
+dbpedia:ontology/wikiPageRevisionID [http]
+dbpedia:ontology/wikiPageExternalLink [http]
+foaf:name [http]
+foaf:homepage [http]
+dbpedia2:name [http]
+foaf:depiction [http]
+<http://purl.org/dc/terms/description> [http]
+foaf:givenName [http]
+foaf:gender [http]
+foaf:isPrimaryTopicOf [http]
+foaf:surname [http]
+dbpedia:ontology/thumbnail [http]
+
+...
+
+__________________________________________________________________
+
+
+2- Get all the properties, except rdf:type, 
+that can be applied to instances of the Politician class
+
+select  ?properties where { 
```

Comment:
Same here. And I assume that everywhere else. Minor change, but important in order to avoid duplications.